From: Thomas Bogendoerfer <tbogendoerfer@suse.de>
Subject: kabi protect struct vf_info
Patch-mainline: never, kabi
References: bsc#1136345 jsc#SLE-4681

Upstream commit 8b965f3f649c ("cxgb4: Add VF Link state support")
adds a new field link_state to struct vf_info.h. Hide this new
field from kABI checker.

Signed-off-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/net/ethernet/chelsio/cxgb4/cxgb4.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/net/ethernet/chelsio/cxgb4/cxgb4.h
+++ b/drivers/net/ethernet/chelsio/cxgb4/cxgb4.h
@@ -885,7 +885,9 @@ struct vf_info {
 	unsigned int tx_rate;
 	bool pf_set_mac;
 	u16 vlan;
+#ifndef __GENKSYMS__
 	int link_state;
+#endif
 };
 
 enum {
